packages/element/src/index.native.js

```diff
@@ -51,3 +56,34 @@ export { serialize as renderToString };
  * @return {WPElement} Dangerously-rendering element.
  */
 export { RawHTML };
+
+/**
+ * Composes multiple higher-order components into a single higher-order component. Performs right-to-left function
+ * composition, where each successive invocation is supplied the return value of the previous.
+ *
+ * @param {...Function} hocs The HOC functions to invoke.
+ *
+ * @return {Function} Returns the new composite function.
+ */
+export { flowRight as compose };
```

Comment:
`compose` and `createHigherOrderComponent` were moved to `compose` package.

Reply:
I think we are still some commits behind in terms of a branch that is compatible with RN.